extensions/awt/deployment/src/main/java/io/quarkus/awt/deployment/AwtProcessor.java

```diff
@@ -258,18 +249,11 @@ JniRuntimeAccessBuildItem setupJava2DClasses(NativeImageRunnerBuildItem nativeIm
         // A new way of dynamically loading shared objects instead
         // of baking in static libs: https://github.com/oracle/graal/issues/4921
         classes.add("sun.awt.X11FontManager");
-        if (v.javaVersion.feature() != 19) {
-            classes.add("java.awt.GraphicsEnvironment");
-            classes.add("sun.awt.X11GraphicsConfig");
-            classes.add("sun.awt.X11GraphicsDevice");
-            classes.add("sun.java2d.SunGraphicsEnvironment");
-            classes.add("sun.java2d.xr.XRSurfaceData");
-        }
```

Comment:
@Karm these are cleanup, as we no longer support JDK 19, so I always include these classes